src/openrct2/ride/TrackDesign.h

```diff
@@ -191,6 +191,9 @@ enum
     MAZE_ELEMENT_TYPE_EXIT = (1 << 7)
 };
 
+// Used for the track preview in the Track Designs window, but not for the ghost shown when placing one.
```

Comment:
Can you remove this comment again? I know I asked for it, but on second glance, it seems like it’s not the case after all. (Which is also why I removed  the line note in which I asked for it.)